control_panel_api/tools.py

```diff
@@ -0,0 +1,66 @@
+import secrets
+
+from django.conf import settings
+from rest_framework.exceptions import APIException
+
+from control_panel_api.helm import Helm
+
+
+SUPPORTED_TOOL_NAMES = [
+    'rstudio',
+]
```

Comment:
Should this be in config, or somewhere not in the code? (DB?)

Reply:
I think we'll iterate on this as we'll introduce more tools.
For now my preference is for this list to be close to the code which uses it.

Reply:
👍